src/core/continuous_space.jl

```diff
@@ -178,6 +178,23 @@ function move_agent!(agent::A, model::ABM{A, S, F, P}, dt = 1.0) where {A<:Abstr
   return agent.pos
 end
 
+"""
+    move_agent!(agent::A, model::ABM{A, ContinuousSpace}, vel::NTuple{D, N}, dt = 1.0)
+Propagate the agent forwards one step according to `vel` and the model's space.
+
+For this continuous space version of `move_agent!`, the "evolution algorithm"
+is a trivial Euler scheme with `dt` the step size, i.e. the agent position is updated
+as `agent.pos += vel * dt`.
+"""

```

Comment:
I don't think this information should be repeated.

Reply:
Duplicating information leads to less clear docs. It might sound counter intuitive, but it is in fact counter productive to say the same things again and again.